skbio/stats/_subsample.py

```diff
@@ -231,7 +231,7 @@ def subsample_counts(counts, n, replace=False):
         raise ValueError("Only 1-D vectors are supported.")
 
     counts_sum = counts.sum()
-    if n > counts_sum:
+    if n > counts_sum and not replace:
```

Comment:
Thanks for the fix! Can you update the `ValueError` below to indicate the issue applies to subsampling without replacement? Also can you update line 176 in the docstring above that describes when this `ValueError` is raised?